src/test.js

```diff
@@ -732,18 +732,13 @@ function internalStart( test ) {
 	}
 }
 
-function numberOfTests( module ) {
-	let count = module.tests.length,
-		modules = [ ...module.childModules ];
+function numberOfTests( module, offset = 0 ) {
```

Comment:
I have no problem with changing the order of these arguments to make everything cleaner:

```js
function numberOfTests( offset, module ) {
  return module.childModules.reduce( numberOfTests, module.tests.length + offset);
}
```

If we don't change the order, let's using something more descriptive than `p` for the callback arg.